js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+        ];
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (symbols !== undefined) {
+            request['symbol'] = symbols.join (',');
+        }
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchTickers', undefined, params);
+        let tickers = undefined;
+        if (marketType === 'spot') {
+            tickers = await this.spotPublicGetTicker24hr (this.extend (request, query));
+            //
+            // (Note: for single symbol, only one object is returned, instead of array)
+            //
+            //     [
+            //       {
+            //         "symbol": "BTCUSDT",
+            //         "priceChange": "184.34",
+            //         "priceChangePercent": "0.00400048",
+            //         "prevClosePrice": "46079.37",
+            //         "lastPrice": "46263.71",
+            //         "lastQty": "",
+            //         "bidPrice": "46260.38",
+            //         "bidQty": "",
+            //         "askPrice": "46260.41",
+            //         "askQty": "",
+            //         "openPrice": "46079.37",
+            //         "highPrice": "47550.01",
+            //         "lowPrice": "45555.5",
+            //         "volume": "1732.461487",
+            //         "quoteVolume": null,
+            //         "openTime": 1641349500000,
+            //         "closeTime": 1641349582808,
+            //         "count": null
+            //       }
+            //     ]
+            //
+        } else if (marketType === 'swap') {
+            const response = await this.contractPublicGetTicker (this.extend (request, query));
+            //     {
+            //         "success":true,
+            //         "code":0,
+            //         "data":{
+            //             "symbol":"ETH_USDT",
+            //             "lastPrice":3581.3,
+            //             "bid1":3581.25,
+            //             "ask1":3581.5,
+            //             "volume24":4045530,
+            //             "amount24":141331823.5755,
+            //             "holdVol":5832946,
+            //             "lower24Price":3413.4,
+            //             "high24Price":3588.7,
+            //             "riseFallRate":0.0275,
+            //             "riseFallValue":95.95,
+            //             "indexPrice":3580.7852,
+            //             "fairPrice":3581.08,
+            //             "fundingRate":0.000063,
+            //             "maxBidPrice":3938.85,
+            //             "minAskPrice":3222.7,
+            //             "timestamp":1634162885016
+            //         }
+            //     }
+            //
+            tickers = this.safeValue (response, 'data', []);
+        }
+        return this.parseTickers (tickers, symbols);
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
```

Comment:
this method is pretty much the same as `fetchTickers` but with a symbols array of length 1. So I think we replace this by:

```Javascript
async fetchTicker (symbol, params = {}) { 
    const symbols = [ symbol ];
   return await this.fetchTickers(symbols, params)
}
```

what do you think?

Reply:
yes, right, they are same endpoints, however because of I was expecting to wait for new swap api, i've not removed the fetchTicker implementation logic, but as noted already , we will not wait for new API, so i'll route them through fetchTickers now.

Reply:
one nuance is that `fetchTickers` (or any plural method) shouldn't have a body where there happens cheching whether one symbol is being called or not (it's not plural methods desired implementation), but in this case I think it will be ok as you say. @kroitor  do you confirm, can I route fT through fTs?

Reply:
I would keep them separate for now because of the defaultType thing in fetchTickers.

Reply:
another nuance is that singular fetchX methods might be provided with `method` prop in params, which means user might need to call specific `method` (i.e. privateGetWhatever) for that singular fetchX method, or even set different `defaultType` under options for that singular method. so in those cases, singular method should have it's own implementation. 
but, again, in this particular case, there seems no such issue, and can be routed.